src/main/main.cpp

```diff
@@ -1,6 +1,7 @@
 #include "main.h"  // IWYU pragma: keep // due to convention
 
 bool isServerRunning_g = true;
+std::string serverHostname_g;
```

Comment:
string is appropriate due to being c++ and getting automatically deleted when out of scope